src/components/LinkRenderer/LinkRenderer.tsx

```diff
@@ -1,17 +1,44 @@
 import { memo } from 'react';
+import s from './LinkRenderer.module.css';
 
-export const LinkRenderer = memo(function (props: any) {
+// types expected by react-markdown library
+type ElementWrapProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+>;
+
+export const LinkRenderer = memo(function (props: ElementWrapProps) {
   return (
-    <a
-      href={props.href}
-      target="_blank"
-      rel="noreferrer"
-      onClick={stopPropagation}
-    >
+    <a href={props.href} target="_blank" rel="noreferrer" onClick={stopPropagation}>
       {props.children}
     </a>
   );
 });
+
+// 192 and 12 are the values proposed by the task #8798
+export function getShortLinkRenderer(
```

Comment:
oooh, it looks so verbose 😞 

I suggest next: 
- as for now we are ok with default options we use it like it was 
```
 <ReactMarkdown
                components={{ a: ShortLinkRenderer }}
 ```
 in case we need to pass options 
 ```
 components={{ a: (props) => <ShortLinkRenderer maxWidth={100} {...props} /> }}
 ```
 then we can use component without wrapper 
 ```
 type ShortLinkRenderer = ElementWrapProps & {
  maxWidth?: number;
  truncateAmount?: number;
};

export function ShortLinkRenderer({
  children: linkArr,
  maxWidth = 192,
  truncateAmount = 12,
}: ShortLinkRenderer) {
 ```